lib/ex_unit/lib/ex_unit/assertions.ex

```diff
@@ -123,10 +123,8 @@ defmodule ExUnit.Assertions do
         end
       end)
 
-    quote do
-      right = unquote(right)
-      expr  = unquote(code)
-      unquote(vars) =
+    match_case =
```

Comment:
In general I rather name things for their intention over what they happen to be. In this case the intention is to do pattern matching and using "case" could be regarded as an is implementation detail. Don't feel strongly about it though, If the variable assignment is only two lines away naming is less important.